docs/security/security-overview.md

```diff
@@ -156,7 +156,11 @@ Fabric data resiliency ensures that your data is available if there is a disaste
 
 As an [administrator in Fabric](../admin/admin-overview.md), you get to control capabilities for the entire organization. Fabric enables delegation of the admin role to capacities, workspaces, and domains. By delegating admin responsibilities to the right people, you can implement a model that lets several key admins control general Fabric settings across the organization, while other admins who are in charge of settings related to specific areas.
 
-Using various tools, admins can also [monitor](../admin/admin-overview.md#monitor) key Fabric aspects such as capacity consumption. You can also [view audit logs](../admin/track-user-activities.md) to monitor user activities and investigate unexpected incidents if needed.
+Using various tools, admins can also [monitor](../admin/admin-overview.md#monitor) key Fabric aspects such as capacity consumption.
+
+## Audit Logs
```

Comment:
Why are you updating the security doc instead of the Fabric audit log docs like https://learn.microsoft.com/en-us/fabric/admin/track-user-activities or https://learn.microsoft.com/en-us/fabric/admin/service-admin-portal-audit-logs?

Reply:
We're also updating the audit log docs - that's a separate effort. Good call!